database/postgreSQL/schema.sql

```diff
@@ -0,0 +1,43 @@
+
+DROP TABLE IF EXISTS businesses;
+CREATE TABLE  businesses (
+  id integer NOT NULL,
+  name character varying(100),
+  PRIMARY KEY (id)
+);
+
+DROP TABLE IF EXISTS users;
+CREATE TABLE users (
+  id integer NOT NULL ,
+  profilename character varying(50) NOT NULL,
+  profileurl character varying(100),
+  reviewcount integer,
+  friendcount integer,
+  PRIMARY KEY (id)
+);
+
+DROP TABLE IF EXISTS images;
+CREATE TABLE images (
+  id integer NOT NULL,
```

Comment:
Are you sure you don't want the images to be auto-incremented (serial) instead of integers? This would make it easier for when you do a POST request and add in some new photos.